src/main/webapp/js/network.js

```diff
@@ -0,0 +1,173 @@
+    const SOURCE = 0;
+    const SHAPE = "dot"
+    const SHAPE_SIZE = 30;
+    const FONT_SIZE = 32;
+    const BORDER_WIDTH = 2;
+    const WHITE_COLOR = "#ffffff";
+  
+    // initialize nodes and edges arrays
+    var nodes = new Array();
+    var edges = new Array();
+
+    populateGraph();
+    createNetwork();
+
+    /* Populate nodes and edges with the following formatting:
+     ** NODE: {id: ID, label: LABEL, group: GROUP} 
+     ** EDGE: {from: SOURCE, to: DESTINATION}
+    */
+    function populateGraph(){
+      // add source node
+      nodes[SOURCE] = {id: SOURCE, label: "My Computer", group: SOURCE}; 
+
+      // initialize edge counter
+      var ed = 0;
+
+      // add all destinations
+      for (var i = 1; i < data.length; i++) {
+        nodes[i] = { id: i, label: data[i].destination, group: i};
+
+        // add edges based on freqs
+        for (var j = 0; j < data[i].frequency; j++) {
+          edges[ed] = {from: SOURCE, to: i};
+          ed++;
+        }
+      }
+    }
+
+    /* Initialize network based on nodes and edges. */
+    function createNetwork(){
+      var container = document.getElementById("mynetwork");
+
+      var finalData = {
+        nodes: nodes,
+        edges: edges,
+      };
+
+      var options = {
+        nodes: {
+          shape: SHAPE,
+          size: SHAPE_SIZE,
+          font: {
+            size: FONT_SIZE,
+            color: WHITE_COLOR,
+          },
+          borderWidth: BORDER_WIDTH,
+        },
+        edges: {
+          width: BORDER_WIDTH,
+        },
+      };
+
+      network = new vis.Network(container, finalData, options);
+    }
+  });
+}
+
+
+/** Create network graph two with hard-coded data. */
+function createNetworkTwo(){
+  const SOURCE = 0;
+  const SHAPE = "dot"
+  const SHAPE_SIZE = 30;
+  const FONT_SIZE = 32;
+  const BORDER_WIDTH = 2;
+  const WHITE_COLOR = "#ffffff";
+
+  // initialize nodes and edges arrays
+  var nodes = new Array();
+  var edges = new Array();
+
+  populateGraph();
+  createNetwork();
+
+  /** Populates graph with hard-coded data. */
+  function populateGraph(){
+    nodes = [
+      { id: 0, label: "My Computer", group: 0 },
+      { id: 4, label: "Class C", group: 6 },
+      { id: 7, label: "Class A", group: 8 },
+      { id: 13, label: "Class E", group: 7 },
+      { id: 15, label: "Class D", group: 5 },
+      { id: 16, label: "224.0.0.0", group: 5 },
+      { id: 17, label: "224.0.0.1", group: 5 },
+      { id: 18, label: "193.0.1.1", group: 6 },
+      { id: 19, label: "192.0.1.1", group: 6 },
+      { id: 21, label: "240.0.0.0", group: 7 },
+      { id: 22, label: "241.0.0.0", group: 7 },
+      { id: 23, label: "242.0.0.0", group: 7 },
+      { id: 24, label: "1.0.0.1", group: 8 },
+      { id: 25, label: "1.1.1.1", group: 8 },
+      { id: 26, label: "126.1.1.1", group: 8 },
+      { id: 27, label: "128.2.0.1", group: 9 },
+      { id: 28, label: "Class B", group: 9 },
+      { id: 29, label: "128.1.0.1", group: 9 },
+    ];
```

Comment:
I am not sure how you are going to do this grouping with real data. I believe you have some ideas?